source/refunding_payments/index.html.md.erb

```diff
@@ -252,7 +261,7 @@ Example search response:
     {
       "refund_id": "efj6834secviyyxe8vs861pg3j",
       ...
-    }    
+    }
```

Comment:
I don't know what this is, how I did it, how to undo it, or whether it even makes a difference O_O

Reply:
Your editor removed whitespace, there were a load of spaces after that `}`, this is fine